src/handle_action.cpp

```diff
@@ -1784,6 +1784,15 @@ bool game::handle_action()
                 ui::omap::display();
                 break;
 
+            case ACTION_SKY:
+                if ( m.is_outside( u.pos() ) ) {
```

Comment:
```suggestion
                if( m.is_outside( u.pos() ) ) {
```

Reply:
I'll try not to forget running `make astyle`, thank you. Will be added in the next commit.